docs/cli/makei.md

```diff
@@ -140,6 +140,14 @@ It is likely that the same destination directory will contain converted members
 
   An optional CCSID for the SRC-PF. See the description above for more details.
 
+- **-l, --tolower**
+
+  The generated source file name will be in lowercase.
+
+- **-t, --text**
+
+  The generated source file will include the member text as a comment.
```

Comment:
The generated source file will include the member text as a comment. Note that besides the EBCDIC CCSID, the member text is the only other piece of metadata associated with a SRC-PF member outside of its contents.  Build tools like BOB and Arcad Builder know how to extract this text description and specify it on the appropriate compile command so that the resultant object has the same description.